### PR TITLE
Changed site background in light theme to #FFFFFF

### LINKED ISSUE #1991: Change site background in light theme to White
**What would you like to be added**:
A white background
**Why would you like this to be added**:
Since white is the new grey

### PR DESCRIPTION
**Pull request by tonyvugithub**
## Issue This PR Addresses
Fixed #1991 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description
Changed the background of site in light theme to #FFFFFF instead of #E5E5E5.

### CODE REVIEW

File: src/web/src/theme/index.ts [hunk 1]
```diff
@@ -21,15 +21,15 @@ export const lightTheme: Theme = createMuiTheme({
...
+      default: '#FFFFFF',
       paper: '#d1d1d1',
     },
     text: {
       primary: '#000000',
       secondary: '#121D59',
     },
     action: {
-      selected: '#ffffff',
+      selected: '#FFFFFF',
```
Comment (humphd) [inline src/web/src/theme/index.ts:32]: What does `action.selected` mean?  When is it used, and should it be different from the regular background or the same?

Comment (tonyvugithub) [reply src/web/src/theme/index.ts:32]: this is due to @izhuravlev commit earlier for the search page. so when you click on a filter, it made the selected filter white box. I was surprised to but did not want to remove it for now.

Comment (izhuravlev) [reply src/web/src/theme/index.ts:32]: Action: selected should be different from the background. Search bar changes it's color on hover to theme.palette.action.selected. 
I added this so we can set up 2 different colours for both modes.

**Review by PedroFonsecaDEV — APPROVED** [review]

File: src/web/src/theme/index.ts (hunk 1 above)
Comment (izhuravlev) [reply src/web/src/theme/index.ts:32]: To make sure everything looks good, click or hover on the search bar in the light mode to see how it looks.

**Review by birtony — COMMENTED** [review]
@PedroFonsecaDEV do we use the upper- or the lowercase notation for hex values? I think we should standardize this and configure Prettier/Eslint accordingly.

**Comment by PedroFonsecaDEV** [comment]
@birtony sounds good. Filed https://github.com/Seneca-CDOT/telescope/issues/1994 to fix this.

**Comment by tonyvugithub** [comment]
@birtony : I think uppercase should be the way to go

**Review by birtony — APPROVED** [review]

**Pull request merged**